### PR TITLE
Remove ncurses usage dependency

### LINKED ISSUE #404: [Runtime Issue]: ./Descent3: /lib64/libncurses.so.6: version `NCURSES6_5.0.19991023' not found (required by ./Descent3)
### Build Version

v1.5 <0ff98a7> and below

### Operating System Environment

- [ ] Microsoft Windows (32-bit)
- [ ] Microsoft Windows (64-bit)
- [ ] Mac OS X
- [X] Linux (specify distribution and version below)

### CPU Environment

- [ ] x86 (32-bit Intel/AMD)
- [X] x86\_64 (64-bit Intel/AMD)
- [ ] ARM (32-bit)
- [ ] ARM64 (64-bit; sometimes called AArch64)
- [ ] Other (RISC V, PPC...)

### Game Modes Affected

- [ ] Single player
- [ ] Anarchy
- [ ] Hyper-Anarchy
- [ ] Robo-Anarchy
- [ ] Team Anarchy
- [ ] Capture the Flag
- [ ] Bounty
- [ ] Entropy
- [ ] Hoard
- [ ] Monsterball
- [ ] Cooperative

### Game Environment

_No response_

### Description

OS: Linux
Distribution: OpenSUSE Tumbleweed

Seems like if i try to run `Descent3` the game does not start. Therefore, starting the game in terminal, the following output is seen:

`./Descent3: /lib64/libncurses.so.6: version `NCURSES6_5.0.19991023' not found (required by ./Descent3)`

I did try to grab `ncurses-devel` but it did not help.

I am unsure if it is a game issue or if it is something on my end.

### Regression Status

_No response_

### Steps to Reproduce

_No response_

### PR DESCRIPTION
**Pull request by winterheart**
## Pull Request Type

- [ ] GitHub Workflow changes
- [ ] Documentation or Wiki changes
- [x] Build and Dependency changes
- [x] Runtime changes
  - [ ] Render changes
  - [ ] Audio changes
  - [ ] Input changes
  - [ ] Network changes
  - [x] Other changes

### Description

Remove ncurses-based type of console, which is enabled only with `-svgalib` and `-dedicated` options. Since there no svgalib direct support (which is pretty old and too specific to Linux tech), it's better completely remove this code for simplicity and reducing external dependencies.

Removed `-svgalib` option as unused now.

### Related Issues

Fixes #404.

### Checklist

- [x] I have tested my changes locally and verified that they work as intended.
- [x] I have documented any new or modified functionality.
- [x] I have reviewed the changes to ensure they do not introduce any unnecessary complexity or duplicate code.
- [x] I understand that by submitting this pull request, I am agreeing to license my contributions under the project's license.

### CODE REVIEW

File: lib/AppConsole.h [hunk 1]
```diff
@@ -42,7 +42,11 @@
 #define APPCONSOLE_H
 
 #define CON_MAX_STRINGLEN 768
+
+bool con_Create(int flags);
```
Comment (Lgt2x) [inline lib/AppConsole.h:46]: are these new declarations also implemented in the Windows build anywhere? If they're not common to both builds, they should probably not go here in a common header

Comment (winterheart) [reply lib/AppConsole.h:46]: Yes, these supposed to go to lnxapp.h... Still, in #410 they will be deleted anyway.

Comment (winterheart) [reply lib/AppConsole.h]: On second thought I decided don't move these declarations at all.

**Review by Lgt2x — APPROVED** [review]
Looks good now, merging

**Pull request merged**